tests/Model/StubsContainer.php

```diff
@@ -126,6 +126,11 @@ public function getFunction($name, $sourceFilePath = null, $shouldSuitCurrentPhp
         if (!empty($functions)) {
             return array_pop($functions);
         }
+        // Added this while tracking down "Attempt to read property "parameters" on null" error.
+        // Happens when BasePHPElement::entitySuitsCurrentPhpVersion returns false when running tests on PHP beta: getenv('PHP_VERSION')
+        if (!$functions) {
+            throw new RuntimeException("Could not get function " . $name . " from reflection, possibly unsupported PHP_VERSION in getenv");
+        }
         return null;
```

Comment:
If we will move check to the `else` branch according to the comment above `return null` could be removed as redundant and return type in declaration also can be changed to non-nullable